extensions/cornerstone/src/init.tsx

```diff
@@ -427,9 +432,9 @@ function _showCPURenderingModal(uiModalService, hangingProtocolService) {
   };
 
   const { unsubscribe } = hangingProtocolService.subscribe(
-    hangingProtocolService.EVENTS.HANGING_PROTOCOL_APPLIED_FOR_VIEWPORT,
-    ({ progress }) => {
-      const done = callback(progress);
+    hangingProtocolService.EVENTS.PROTOCOL_CHANGED,
```

Comment:
The callback is still needed, otherwise it shows the progress bar forever, and we want the progress bar initially.  I hope at some point to get intermediate callbacks with status updates.